README.md

```diff
@@ -1,2 +1,111 @@
 # Capstone_project_1
 First Microverse capstone project (module 1-block 5)
+
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="./images/my_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Your image is not displaying on the README. You can add your own logo or if you don't have one, leave the README without a logo.